INSTRUCTION.md

```diff
@@ -0,0 +1,22 @@
+## How to validate App is running
+``
+     http://localhost:30007/api
+``
+
+## How to validate configMap data is mounted as files in the right order
+## Secret data is mounted as a file
+
+``
+     
+     kubectl get pods
+    kubectl exec {pod} -it -- sh 
+    ls
+    cd configs
+    ls
+    cat APP_ENV
```

Comment:
It would be helpful to clarify what users should expect to see when they `cat APP_ENV`. Providing expected output or a brief explanation would improve the clarity of the instructions.